source/scripting_v3/GTA/DrawBoxFlags.cs

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright (C) 2015 crosire & contributors
+// License: https://github.com/crosire/scripthookvdotnet#license
+//
+
+using System;
+
+namespace GTA
+{
+	[Flags]
+	public enum DrawBoxFlags
+	{
+		OutsideOnly = 1,
+		InsideOnly = 2,
+		BothSides = OutsideOnly & InsideOnly
```

Comment:
Just noticed that this is probably supposed to be `OutsideOnly | InsideOnly`, since otherwise `BothSides` will be zero.